dranspose/replay.py

```diff
@@ -366,4 +370,3 @@ def replay(
             stop_event.wait()
         except KeyboardInterrupt:
             pass
-        logger.info("replay finished")
```

Comment:
This log message is present that if the web server should stay alive, the tester knows when to start probing the results, i.e. all sources are depleted.